src/components/tracked-controls.js

```diff
@@ -84,7 +85,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     var controllerQuaternion = this.controllerQuaternion;
     var controllerEuler = this.controllerEuler;
     var deltaControllerPosition = this.deltaControllerPosition;
-    var hand = (controller ? controller.hand : undefined) || DEFAULT_HANDEDNESS;
+    var hand = this.data.hand || (controller ? controller.hand : undefined) || DEFAULT_HANDEDNESS;
```

Comment:
Yep. this should not be part of the component API. Let's keep the const approach we had before